### PR TITLE
Added Kieker components.

### PR DESCRIPTION
**Pull request by okism**
Added the logo and descriptions for components.

### CODE REVIEW

File: model/components/kieker.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      - kieker-trace-diagnosis-gui
+      - kieker-trace-analysis-ui
+
+# kieker analysis
+  - id: kieker-analysis
+    name: Kieker Analysis
+    description: Kieker Analysis component for reading, analyzing, and visualizing the monitoring data. It manages the lifecycle of various analysis plugins.
+    link: http://kieker-monitoring.net/
+    github:
+      - kieker-monitoring/kieker
```
Comment (AlexanderWert) [inline model/components/kieker.yml]: Do you have additional github repositories for the analysis plugins? if, yes you can add all of them here. The github Property is a list, so you can have multiple per component.

File: model/components/kieker.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+components:
+
+# kieker monitoring
+  - id: kieker-monitoring
```
[the remark this answers is not in the document]
Comment (okism) [reply model/components/kieker.yml]: For the first two points, I agree. However, save for categories, everything else is exactly the same. Seemed redundant, but I'll split.

For the third point: You are right about the Analysis component. On the other hand, in the sense of the definition for the Pipeline components, Kieker's Monitoring Controller component can also do some pre-processing and send the data to different destinations (through different writers).

File: model/components/kieker.yml (hunk 1 above)
Comment (okism) [reply model/components/kieker.yml]: No. It's all a part of this one repository.

Only trace diagnosis GUI is in another repo.

File: model/components/kieker.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (AlexanderWert) [reply model/components/kieker.yml]: Fine for me :+1:
Just update the pull request and I will merge.

Thanks again!

**Pull request merged**